src/typings.ts

```diff
@@ -9,6 +9,7 @@ export interface PrivateConfigsOptions {
   sdkVersion?: string; // semver
 }
 
+export type AuthN = 'passwordless' | 'sso'
```

Comment:
```suggestion
export type AuthNMethod = 'passwordless' | 'sso'
```
Let's not change it to avoid confusion and the need of guessing.